src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -118,6 +120,27 @@
 Get_PublicKey_Info_Fault_Type = 'Error while fetching the PoP publickey information from client proxy'
 PoP_Public_Key_Expried_Fault_Type = 'The PoP public key used to generate the at has expired'
 Post_AT_To_ClientProxy_Failed_Fault_Type = 'Failed to post access token to client proxy'
+Kubectl_Get_Events_Failed = "Error while doing kubectl get events"
+Arc_Agents_Logger_Fault_Type = "Error occured in arc agents logger"
+Arc_Agents_Events_Logger_Fault_Type = "Error occured in arc agents events logger"
+Arc_Deployments_Logger_Fault_Type = "Error occured in deployments logger"
+Agent_State_Check_Fault_Type = "Error ocuured while performing the agent state check"
+Agent_Version_Check_Fault_Type = "Error occured while performing the agent version check"
+Executing_Diagnoser_Job_Fault_Type = "Error while executing Diagnoser Job"
+Diagnoser_Container_Check_Fault_Type = "Error occured while performing the diagnoser container checks"
+Cluster_DNS_Check_Fault_Type = "Error occured while performing cluster DNS check"
+Outbound_Connectivity_Check_Fault_Type = "Error occured while performing outbound connectivity check in the cluster"
+MSI_Cert_Check_Fault_Type = "Error occured while trying to pull MSI certificate"
```

Comment:
this should be - error occurred while trying to perform MSI ceritificate presence check right? not error occured while trying to pull MSI certificate